### PR TITLE
update: Recommend reading the Newsboat Arch Wiki page

### PR DESCRIPTION
**Pull request by rollsicecream**
Changes proposed in this PR:

- update: Recommend reading the Newsboat Arch Wiki page for people who want to use it.

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/news-aggregators.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ A **news aggregator** is software which aggregates digital content from online n
 
 </div>
 
+If you decide to use Newsboat as your news aggregator, we recommend you reading the [Arch Wiki page](https://wiki.archlinux.org/title/Newsboat) about Newsboat as it contains a lot of information about how to use it, customize it and even some tips to use Newsboat efficiently.
```
Comment (jonaharagon) [inline docs/news-aggregators.md:154]: Is this a better resource than the documentation at https://newsboat.org/releases/2.36/docs/newsboat.html?

Comment (rollsicecream) [reply docs/news-aggregators.md:154]: At least, for me it was.

**Comment by dngray** [comment]
Personally I'd like to avoid linking to arch wiki, just as we try to avoid linking to wikipedia.

Some of the articles mentioned there get dated when the single maintainer leaves them alone.

**Comment by kimg45** [comment]
I think it's best to link to the official documentation, as @dngray said third party documentation can get outdated.

**Pull request closed without merging**